ubuntudesign/developer.ubuntu.com
# Switch over to using ubuntudesign.gsa for search

- Use the newly minted ubuntudesign.gsa PyPi module for querying search results
- Remove all the old search code

This should fix all parsing errors, and generally make things more robust.

Resolves #213
Resolves #212

QA
--

Run the site locally, making sure you've got the latest requirements.

Visit both `http://127.0.0.1:8015/search?q=kylin` and `127.0.0.1:8015/search?q=要了解编写HTML5` and check everything looks good in both cases.
